engine/src/main/java/com/datatorrent/stram/client/StramAgent.java

```diff
@@ -421,7 +421,11 @@ private StramWebServicesInfo retrieveWebServicesInfo(String appId)
         permissionsInfo = new JSONObject(IOUtils.toString(is));
       } catch (IOException ex) {
         // ignore if file is not found
-        LOG.info("Exception in accessing permissions.json", ex);
+        if (LOG.isDebugEnabled()) {
```

Comment:
@vrozov By security configuration, you mean kerberos right? Whether security is enabled or not, these permissions do not affect that security. This information is additional acls that are created, stored with the app and interpreted all by the client.